### ISSUE TITLE
No valid versions available for temp-workspace when trying cra-to-nx

### ISSUE THREAD

**Issue by 72gm** [nrwl/add-nx#39]
I get this error when running npx cra-to-nx 

0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   'temp-workspace@latest',
1 verbose cli   '--global',
1 verbose cli   '--prefix',
1 verbose cli   'C:\\Users\\gmiller\\AppData\\Roaming\\npm-cache\\_npx\\239200',
1 verbose cli   '--loglevel',
1 verbose cli   'error',
1 verbose cli   '--json'
1 verbose cli ]
2 info using npm@6.14.15
3 info using node@v14.18.1
4 verbose npm-session 0d3519544f2ec742
5 silly install loadCurrentTree
6 silly install readGlobalPackageData
7 http fetch GET 304 https://registry.npmjs.org/temp-workspace 872ms (from cache)
8 silly fetchPackageMetaData error for temp-workspace@latest No valid versions available for temp-workspace
9 timing stage:rollbackFailedOptional Completed in 1ms
10 timing stage:runTopLevelLifecycles Completed in 895ms
11 verbose type tag
12 verbose stack temp-workspace: No valid versions available for temp-workspace
12 verbose stack     at pickManifest (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-pick-manifest\index.js:39:11)
12 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\pacote\lib\fetchers\registry\manifest.js:24:14
12 verbose stack     at tryCatcher (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\util.js:16:23)
12 verbose stack     at Promise._settlePromiseFromHandler (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\promise.js:517:31)
12 verbose stack     at Promise._settlePromise (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\promise.js:574:18)
12 verbose stack     at Promise._settlePromise0 (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\promise.js:619:10)
12 verbose stack     at Promise._settlePromises (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\promise.js:699:18)
12 verbose stack     at _drainQueueStep (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:138:12)
12 verbose stack     at _drainQueue (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:131:9)
12 verbose stack     at Async._drainQueues (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:147:5)
12 verbose stack     at Immediate.Async.drainQueues [as _onImmediate] (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:17:14)
12 verbose stack     at processImmediate (internal/timers.js:464:21)
13 verbose cwd C:\Dev\NX\nx-monorepo\track-my-appeal
14 verbose Windows_NT 10.0.19042
15 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "temp-workspace@latest" "--global" "--prefix" "C:\\Users\\gmiller\\AppData\\Roaming\\npm-cache\\_npx\\239200" "--loglevel" "error" "--json"
16 verbose node v14.18.1
17 verbose npm  v6.14.15
18 error code ENOVERSIONS
19 error No valid versions available for temp-workspace
20 verbose exit [ 1, true ]

**Comment by wise-introvert** [comment]
Facing this exact issue. Have you found a solution yet @72gm?

**Comment by 72gm** [comment]
@wise-introvert no, I'm afraid not .. i just ended up doing it manually as it wasn't too much hassle for the proof of concept I was doing ... might want to post an issue on the main nrwl/nx repo as no one seems to be actively triaging this repo

**Comment by kerituni12** [comment]
same issue

**Comment by thatCarlyleGuy** [comment]
same issue
